encoders/video/VideoPaddleEncoder/requirements.txt

```diff
@@ -1,5 +1,5 @@
-paddlepaddle
-paddlehub
-Pillow
-six>=1.14.0
-protobuf>=3.11.0
+paddlepaddle==1.8.5
+paddlehub==1.8.3
+Pillow==8.0.1
```

Comment:
Pillow==8.1.0

Reply:
is it an improvement or a requirement? If tests pass we can take in separate PRs to upgrade dependencies, but taking for each executor the newest dependencies is overkilling now

Reply:
Anyone is welcome later to upgrade dependencies, but out of the scope of this PR

Reply:
okay